e2e_tests/package.json

```diff
@@ -5,7 +5,7 @@
     "main": "index.js",
     "scripts": {
         "check": "tsc && prettier --check '**/*.{ts,json,yml}' && tslint --project .",
-        "pretest": "cargo build --bin cnd && tsc",
+        "pretest": "cargo build --bin cnd && cargo build --bin nectar && tsc",
```

Comment:
Back to my other comment, this shows that we need to separate the tests.
One folder for cnd api
One folder for full e2e

If I am changing something on the http api of cnd, I don't want to compile nectar. The CI can do it but I don't want to be forced locally.